src/views/TabView/TabBarBottom.js

```diff
@@ -57,6 +62,31 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
 
   props: Props;
 
+	keyboardDidShowListener: React.EmitterSubscription;
+	keyboardDidHideListener: React.EmitterSubscription;
+
+  state = {
```

Comment:
nit: Can you match the indentation of the source code? The indentation here is off.